modules/explorer/update.go

```diff
@@ -222,6 +220,26 @@ func (e *Explorer) ProcessConsensusChange(cc modules.ConsensusChange) {
 			}
 		}
 
+		// Update stats for according to SiacoinOutputDiffs
+		for _, scod := range cc.SiacoinOutputDiffs {
+			scoid := scod.ID
+			sco := scod.SiacoinOutput
+
+			if scod.Direction == true {
```

Comment:
use `modules.DiffApply` instead of `true` here, to better show the intent (ditto for the other predicate below)